app/views/challenges/show.erb

```diff
@@ -10,8 +10,8 @@
     <h5>End file</h5>
     <pre class="prettyprint"><%= @challenge.output %></pre>
 
-    <h4><a href="#" onclick="$('.diff').toggle(); return false;">View Diff</a></h4>
-    <pre class="prettyprint diff"><%= @challenge.diff %></pre>
+    <h4><a href="#" onclick="toggle('diff'); return false;">View Diff</a></h4>
+    <pre class="prettyprint" id="diff"><%= @challenge.diff %></pre>
```

Comment:
`diff` should be a class, since we're repeating it on the page.

Reply:
I grepped the project and can't see another "diff" (class or id)
Can you show me the file ?

Reply:
`</facepalm>`

Ignore me. I'm reading diff but thinking submission entry. There is only one diff at the top! All good, merging 👍